core/store/src/test/java/org/locationtech/geowave/core/store/api/DataStoreTestSuite.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2013-2019 Contributors to the Eclipse Foundation
+ *
+ * <p> See the NOTICE file distributed with this work for additional information regarding copyright
+ * ownership. All rights reserved. This program and the accompanying materials are made available
+ * under the terms of the Apache License, Version 2.0 which accompanies this distribution and is
+ * available at http://www.apache.org/licenses/LICENSE-2.0.txt
+ */
+package org.locationtech.geowave.core.store.api;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({DataStoreAddTest.class, DataStoreRemoveTest.class})
```

Comment:
can you delete this?  This is unnecessary - I think the maven plugin we setup to actually only run unit tests named *Test which is actually good because otherwise these tests would run twice (once themselves and once when it encounters this suite).  Anyways, better off not to worry about this.

Reply:
Fixed in commit.